### DISCUSSION TITLE
Metadata INSPIRE Monitoring 2024 evaluation

### DISCUSSION THREAD

**Discussion by Vandapico** [INSPIRE-MIF/helpdesk-validator#1142] [Q&A] [answered]
Please confirm that, for Metadata INSPIRE Monitoring 2024 evaluation, will be used **only** the "Common Requirements for ISO/TC 19139:2007 based INSPIRE metadata records", "Conformance Class 1: 'Baseline metadata for data sets and data set series" and "Conformance Class 2b: 'INSPIRE data sets and data set series metadata for Monitoring'".

Thank you.

**Comment by antjacques** [comment]
Hello @Vandapico 

I asked some questions regarding M&R here but no answers yet unfortunately: https://github.com/INSPIRE-MIF/mr-tools/issues/6

**Comment by fabiovinci** [comment, marked as answer]
Dear all,

we confirm that there will be no changes to the Monitoring 2024 compared to previous years.
